vtex-apps/checkout-io-tests
# CHK-333: Tests for Cart

#### What does this PR do?

As title says. Summary of tests:

<img width="449" alt="Screen Shot 2020-09-14 at 17 46 24" src="https://user-images.githubusercontent.com/26108090/93216074-a74b0a00-f73d-11ea-9813-c149db0602a5.png">

Notice that there are no tests simulating API failure, as we currently do not handle those cases **yet**.

#### How to test it?

Switch to `wdsrocha0e2e` workspace, fetch this branch and run `yarn && yarn e2e:run` (or `yarn e2e:open` if you want to watch the scenarios and play around). All tests inside cart directory should pass.

For some reason, `vtex test e2e --token --workspace` is not working, so that's why you should only test this locally.
